CONTRIBUTING.md

```diff
@@ -100,13 +100,32 @@ pytest
 pytest tests/${TARGET_TEST_FILE_NAME}
 ```
 
-## Continuous Integration
+## Continuous Integration and Local Verification
 
 Optuna repository uses GitHub Actions and CircleCI.
 
 Currently, we are migrating to GitHub Actions but still we use CirclCI for a test of `document`
 because it makes it much easier to check built documentation.
 
+### Local Verification
+
+By installing [`act`](https://github.com/nektos/act#installation) and Docker, you can run
+tests written for GitHub Actions locally.
+
+```bash
+JOB_NAME=checks
+act -j $JOB_NAME
+```
+
+Currently, you can run the following jobs:
```

Comment:
> 
> 
> In my env, I can not run `documentation` and `doctest` because the container does not have `make` command.
> 
> ```
> $ act -j doctest
> ...
> ...
> [Sphinx/doctest]   ✅  Success - Install Dependencies
> [Sphinx/doctest] ⭐  Run Run Doctest
> | /github/workflow/4: line 2: make: command not found
> [Sphinx/doctest]   ❌  Failure - Run Doctest
> Error: exit with `FAILURE`: 127
> ```
> 
> https://github.com/nektos/act#runners
> 
> In the above runners, I chosed `Medium Docker Image`. Maybe, that would be the reason.
> Which runner did you use for act?

As per this, let's make this a little bit more informative.

```suggestion
Currently, you can run the following jobs: `documentation` and `doctest` may not be executable depending on your choice of docker image of act.
```

Reply:
Thank you for the suggestion! It would be more kind to users.